avm/ptn/authorization/pim-role-assignment/modules/definitions.bicep

```diff
@@ -0,0 +1,95 @@
+// ================ //
+// Definitions      //
+// ================ //
+
+@export()
```

Comment:
This file unfortunately a bit of an anti pattern as no one could actually import these types from the module once published as they're not implemented in a file that is published (which could only be the primary `main.bicep` file. As this may sound very confusion, I suggest we have a chat.

Reply:
Resolved